### DISCUSSION TITLE
Why are the east and west wind icons reversed?

### DISCUSSION THREAD

**Discussion by bend-n** [ryanoasis/nerd-fonts#1162] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/70787919/230706744-784caff0-1478-4099-8c66-843b40fc4974.png)

**Comment by Finii** [comment, marked as answer]
What is exactly reversed here?
What would you expect?

---

https://en.wikipedia.org/wiki/East_wind
> An east wind is a [wind](https://en.wikipedia.org/wiki/Wind) that originates in the [east](https://en.wikipedia.org/wiki/East) and blows in a westward direction.

Typical 'map':
```
   N
W     E
   S
```

Compare all directions:

![image](https://user-images.githubusercontent.com/16012374/230707043-da751375-2d66-4ffb-952b-805605bc7ecd.png)

----

Side note:

Why are all your digits broken??

![image](https://user-images.githubusercontent.com/16012374/230707195-70d92f29-70ee-4f3a-af85-b9b5db2bc918.png)

**Reply by bend-n** [reply]
I really dont know why my digits are broken, its kinda annoying.

I would think that the arrow for west wind would point toward the west, no?

**Reply by Finii** [reply]
The arrow shows the direction of the air, so west wind: arrow from west (left) to east (right).
Compare the north-wind and all other icons.

**Reply by bend-n** [reply]
Hmm, I suppose it makes sense.

**Reply by Finii** [reply]
With wind directions you do not point against the wind to where-the-wind-comes-from but you point with the wind in the wind destination direction. Albeit the NAME is from the wind origin :grimacing: Compare Wikipedia entry linked above.
